applications/FluidDynamicsApplication/tests/cfl_output_process_test.py

```diff
@@ -0,0 +1,67 @@
+import KratosMultiphysics as km
+import KratosMultiphysics.FluidDynamicsApplication as kfd
+
+from fluid_analysis_without_solution import FluidAnalysisWithoutSolution
+
+import KratosMultiphysics.KratosUnittest as UnitTest
+import KratosMultiphysics.kratos_utilities as kratos_utilities
+
+class CFLOutputProcessTest(UnitTest.TestCase):
+
+    def setUp(self):
+        # Set to true to get post-process files for the test
+        self.print_output = True
+
+    def testCFLOutput(self):
```

Comment:
Not sure if this test has to be skip if the `StatisticsApplication` is not compiled.

Reply:
Oh sorry,. I forgot that

Reply:
I recommend checking https://github.com/KratosMultiphysics/Kratos/pull/6746 for that ;)

Reply:
:+1: